snippets/lvms-scaling-up-storage-lvmcluster-cr-snippet.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: SNIPPET
```

Comment:
Nit - per our [docs guidelines on snippet metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata), you'd want to keep track of where the snippet is included just like you do with modules:
```suggestion
// Text snippet included in the following modules:
//
// * modules/lvms-scaling-storage-of-clusters-using-cli.adoc
// * modules/lvms-scaling-storage-of-clusters-using-web-console.adoc

:_mod-docs-content-type: SNIPPET
```

Reply:
Updated as per the suggestion.